cv32e40s/tests/programs/custom/illegal_access_loop_tests/illegal_custom_loop_gen.py

```diff
@@ -0,0 +1,65 @@
+"""
+**
+** Copyright 2022 OpenHW Group
+**
+** Licensed under the Solderpad Hardware Licence, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+** https://solderpad.org/licenses/
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+**
+*******************************************************************************
+**
+** Script which generates the usermode custom instructions, to check illegal access. Instructions are written directly to the 'illegal_custom_loop.S' file after the 'input_string' declared below.
+**
+*******************************************************************************
+"""
+
+func6 = {35, 51} # usermode unprivileged function codes
+outer_loop = 2048 # 11-bit custom instruction
+inner_loop = 32 # 5-bit custom instruction
```

Comment:
What is "5-bit custom instruction"?

Reply:
Added more comments to expand on this. The 11-bit and 5-bit refers to the bit-fields that hold the custom instructions.